spec/forms/new_project_form_spec.rb

```diff
@@ -42,7 +42,9 @@
     end
 
     it "returns false with an error added when the CSV fails to parse" do
-      allow_any_instance_of(RemoteCsv)
```

Comment:
Do we know why `allow_any_instance_of` is not recommended? it seems cleaner than stubbing 'new'?

Reply:
Sure, there's a description on RSpec Style Guide: https://rspec.rubystyle.guide/#any_instance_of

`allow_any_instance` is quite controversial in the RSpec world, with [RSpec themselves](https://relishapp.com/rspec/rspec-mocks/docs/working-with-legacy-code/any-instance) advising against using it unless dealing with legacy code.

I've dealt with the warnings in the more bone headed way here of just fixing the tests like for like, so I imagine a more thorough refactor could do nicer things with them.